### PR TITLE
Adding Security RP API swagger

### PR DESCRIPTION
**Pull request by chlahav**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by chlahav** [comment]
@sergey-shandar continuing PR

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
There are linter and model validation errors that need to be fixed.  Please look at the travis log [here](https://travis-ci.org/Azure/azure-rest-api-specs/builds/379952064?utm_source=github_status&utm_medium=notification), specifically see the tasks "MODE=linter PR_ONLY=true" and "MODE=model PR_ONLY=true".

**Comment by sergey-shandar** [comment]
Investigating a problem with `$id` and `$ref` usage in examples.

File: specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/examples/Alerts/GetAlertsSubscriptionsLocation_example.json [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                                "reason": "Some user reason"
+                            }, {
+                                "type": "Process",
+                                "reason": "Some proccess reason"
+                            }, {
+                                "type": "Computer",
+                                "reason": "Some computer reason"
+                            }],
+                            "canBeInvestigated": true,
+                            "entities": [{
```
Comment (sergey-shandar) [inline specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/examples/Alerts/GetAlertsSubscriptionsLocation_example.json]: `AlertProperties` has no `entities`, `canBeInvestigated`,  `confidenceReasons`, `confidenceScore`, `workspaceArmId`, `instanceId`, `subscriptionId` properties.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

**Comment by chlahav** [comment]
it seems that the PR was closed by mistake and not it doesn't detect any changes.
is there a way to re-open it?

**Pull request closed without merging**